README.md

```diff
@@ -1,24 +1,66 @@
-# README
+# Disaster Relief For Immigrants - DRAI (www.ca-drai.org)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is a tool designed for Assisters/Supervisors of community based organizations(CBOs), and will not be directly accessed by clients.
+This is the production code for the ca-drai.org service.
 
-Things you may want to cover:
+## Installation
 
-* Ruby version
+### Setup
 
-* System dependencies
+Run [`bin/setup`](bin/setup).
 
-* Configuration
+### Common Issues
 
-* Database creation
+-  If setting up Postgres.app, you will also need to add the binary to your path. e.g. Add to your `~/.zshrc`: `export PATH="$PATH:/Applications/Postgres.app/Contents/Versions/latest/bin"`
 
-* Database initialization
+#### Dependencies
 
-* How to run the test suite
+_You may already have these installed. Try running the **Setup** instructions above first._
 
-* Services (job queues, cache servers, search engines, etc.)
+1. [Homebrew](https://brew.sh/) for MacOS.
+2. A ruby version manager like [RVM](https://github.com/codeforamerica/howto/blob/master/Ruby.md) or rbenv.
+3. Locally installed Ruby of the version defined in `.ruby-version`. Because we are using an older version of Ruby, you may also need to run `gem update --system` before running `gem install bundler` and `bundle install`.
+4. Postgres 9.4+ `brew install postgres` or [Postgres.app](https://github.com/codeforamerica/howto/blob/master/PostgreSQL.md).
+5. Install Aptible CLI with Homebrew by running `brew cask install aptible`
+6. Install system dependencies defined in [Brewfile](Brewfile) with `brew bundle`
+7. Chrome and possibly Firefox browsers
 
-* Deployment instructions
+## Local update
 
-* ...
+1. Run `bin/update`
+
+## Testing
+
+- Test suite: `bin/rspec`. For more detailed logging use `bin/rspec LOUD_TESTS=true`.
+
+## Writing code
+
+Learn more about our engineering team processes in
+[contributing.md](contributing.md).
+
+## Pushing code
+
+If pairing, use [git-duet](https://github.com/git-duet/git-duet). (This will be installed when running `brew bundle`.)
+
+## Deploy
+
+Deploy runs automatically upon pushing to master.
+
+## Accessing the production console
+
+Run `aptible ssh --environment drai-production --app rails`
+Then run `bin/rails c`
+
+Use caution as the database is writable in the console.
+
+## Infrastructure
+
+### Overview
+
+- Hosted on Aptible
```

Comment:
I'm not super familiar with Aptible and it's infrastructure so I wasn't too sure what to put here.

Reply:
Good question. I think you could document the services that are necessary to be run in production:

- `$ bin/rails server`: the webserver
- `$ bin/rails jobs:work`: the jobworker (delayed_job)
- `$ bin/scheduler`: the cron replacement

And also the service dependencies, of which the only one is Postgres.